aiohomekit/controller/ble/bleak.py

```diff
@@ -163,9 +147,16 @@ async def get_characteristic_iid(
         self._iid_cache[char] = iid
         return iid
 
-    @property
+    @cached_property
     def mtu_size(self) -> int:
         """Return the mtu size of the client."""
+        # Avoid bluezdbus/client.py:561: UserWarning: Using default MTU value. Call _acquire_mtu() or set _mtu_size first to avoid this warning.
```

Comment:
```suggestion
        # Avoid bluezdbus/client.py:561: UserWarning: Using default MTU value. 
        # Call _acquire_mtu() or set _mtu_size first to avoid this warning.
```